src/main/java/com/telran/libraryapp/entity/Author.java

```diff
@@ -19,8 +24,17 @@ public class Author {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
+    @NotNull
+    @Length(max=45,message = "{validation.author.name}")
+    @Pattern(regexp = "[A-Za-z\\s.]{1,45}", message = "{validation.author.name}")
     private String name;
+
+    @Length(max=45,message = "{validation.author.name}")
+    @Pattern(regexp = "[A-Za-z\\s.]{0,45}", message = "{validation.author.surname}")
```

Comment:
Plase check and test whether the current names and surnames in data.sql can be updated with this regex. 
" - ", " ' " should be included in regex